README.md

```diff
@@ -1,35 +1,52 @@
 # QuBaLab
 
-This is a Python package for exploring quantitative bioimage analysis... *especially* (but not exclusively) in
+This is a Python package for exploring quantitative bioimage analysis...
+*especially* (but not exclusively) in
 combination with QuPath (https://qupath.github.io/).
 
 The name comes from **Quantitative Bioimage Analysis Laboratory**.
-This is chosen to be reminiscent of QuPath (*Quantitative Pathology*), but recognizes that neither is really restricted
-to pathology.
+This is chosen to be reminiscent of QuPath (*Quantitative Pathology*),
+but recognizes that neither is really restricted to pathology.
 
 ## Goals
 
 QuBaLab isn't QuPath - they're just good friends.
 
-* **QuPath** is a user-friendly Java application for bioimage analysis, which has some especially nice features for
-handling whole slide and highly-multiplexed images. But lots of bioimage analysis researcher is done in Python,
-and is hard to integrate with QuPath.
-* **QuBaLab**'s main aim is to help with this, by providing tools to help exchange data between QuPath and Python
-*without any direct dependency on QuPath and Java*. It therefore doesn't require QuPath to be installed, and
-can be used entirely from Python.
-
-QuBaLab doesn't share code with QuPath, but is uses many of the same conventions for accessing images and
-representing objects in a GeoJSON-compatible way.
-By using the same custom fields for things like measurements and classifications, exchanging data is much easier.
+* **QuPath** is a user-friendly Java application for bioimage analysis,
+    which has some especially nice features for
+    handling whole slide and highly-multiplexed images.
+    But lots of bioimage analysis research is done in Python,
+    and is hard to integrate with QuPath.
+* **QuBaLab**'s main aim is to help with this, by providing tools to help
+    exchange data between QuPath and Python
+    *without any direct dependency on QuPath and Java*.
+    It therefore doesn't require QuPath to be installed, and
+    can be used entirely from Python.
+
+QuBaLab doesn't share code with QuPath, but is uses many of the same conventions
+for accessing images and representing objects in a GeoJSON compatible way.
+By using the same custom fields for things like measurements and classifications,
+exchanging data is much easier.
+
+[paquo](https://paquo.readthedocs.io/en/latest/index.html) is an existing library
```

Comment:
Proposed replacement section:

---

### How does QuBaLab compare to paquo?

[paquo](https://paquo.readthedocs.io) is an existing library linking Python and QuPath that provides a pythonic interface to QuPath.

_We think paquo is great, and don't want to replace it!_

Here are the 3 main differences as we see them:
1. **Target audience**
    * paquo is written mostly for Python programmers who need to work with QuPath data
    * QuBaLab is written mostly for QuPath users who want to dip into Python
2. **Convenience vs. Efficiency**
    * paquo is based on [JPype](https://jpype.readthedocs.io/) to provide full & efficient access to Java from Python
    * QuBaLab is based on [Py4J](https://www.py4j.org) to exchange data between Java & Python - preferring convenience over efficiency
3. **Pixel access**
    * paquo is for working with QuPath projects and objects - accessing pixels is [beyond its scope (at least for now)](https://forum.image.sc/t/paquo-read-write-qupath-projects-from-python/41892/26)
    * QuBaLab enables requesting pixels as numpy or dask arrays, and provides functions to convert between thresholded images & QuPath objects

So if you're a Python programmer who needs an intuitive and efficient way to work with QuPath data, use paquo.

But if you're a QuPath user who wants to switch to Python for some tasks, including image processing, you might want to give QuBaLab a try.